Tasks/SonarQubePostTest/SummaryReport/QualityGateSection.ps1

```diff
@@ -0,0 +1,222 @@
+
+function GetQualityGateSectionContent
+{
+    Write-Verbose "Formatting the quality gate report section"
+    
+    WaitForAnalysisToFinish
+    
+    $qualityGateResponse = FetchQualityGateDetails
+    $qualityGateStatus = $qualityGateResponse.projectStatus.status
+    $section1 = FormatQualityGateStatusSection $qualityGateStatus
+    
+    $warningsAndErrors = GetQualityGateWarningsAndErrors $qualityGateResponse    
+    $section2 = FormatQualityGateMessagesSection $warningsAndErrors
+    
+    $content = FormatQualityGateSection $section1 $section2
+    return $content
+}
+
+function FormatQualityGateSection
```

Comment:
I will rename section1 and section2 to qualityGateStatusSection and qualityGateMessagesSection